### PR TITLE
update on installation process on termux

### PR DESCRIPTION
**Pull request by sakarimov**
the installation process on termux should be updated, as written in it's documentation https://github.com/termux/termux-packages/wiki/Termux-file-system-layout, the ```$PREFIX``` will immediately refer to ```/data/data/com.termux/files/usr```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ more information in their docs.
 ```bash
 git clone https://github.com/OmarMohamedKhallaf/Timewarrior-Pomodoro.git pomo
 cd pomo
-# Use -DCMAKE_INSTALL_PREFIX=$PREFIX/usr for termux on android
+# Use -DCMAKE_INSTALL_PREFIX=$TERMUX_PREFIX/usr for termux on android
```
Comment (omar-khallaf) [inline README.md:24]: You're right, but shouldn't it be `-DCMAKE_INSTALL_PREFIX=$PREFIX`?
`$TERMUX_PREFIX` isn't defined in my environment.

**Review by omar-khallaf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sakarimov) [reply README.md:24]: as written in the documentation, ```$TERMUX_PREFIX = /data/data/com.termux/files```, and ```$PREFIX = /data/data/com.termux/files/usr```, i'm not very familiar with termux actually, i don't know how is the most common variable used, therefore i only add ```$TERMUX_PREFIX```, or should it be written like this?

```# Use -DCMAKE_INSTALL_PREFIX=$PREFIX or $TERMUX_PREFIX/usr for termux on android```

Comment (omar-khallaf) [reply README.md:24]: I didn't read well.
`$TERMUX_PREFIX` seems to be defined in the context of packaging, so for this manual install I think
`-DCMAKE_INSTALL_PREFIX=$PREFIX` is the right choice.

Comment (sakarimov) [reply README.md:24]: i see, good choice

Comment (omar-khallaf) [reply README.md:24]: Wait you mentioned something important it should be just `$PREFIX` not `$PREFIX/usr` if you don't mind you can update it or I will do it later.
Thanks again.

**Pull request closed without merging**